src/main/java/pages/account/AccountForm.java

```diff
@@ -0,0 +1,262 @@
+package pages.account;
+
+import framework.DriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.CacheLookup;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import pages.basepages.FormBase;
+
+/**
+ * Created by noelia on 8/26/2015.
+ */
+public class AccountForm extends FormBase {
+
+    @FindBy(id = "acc2")
+    @CacheLookup
+    private WebElement accountNameTxt;
+
+    @FindBy(id = "acc5")
+    @CacheLookup
+    private WebElement accountNumber;
```

Comment:
In the same a suffix should be added to know what kind of html component it is. For example: 
AccountNumberTxt
....CombBox
...TxtArea
....Icon
....Img

Please add those suffixes to all your WebElements.